### ISSUE TITLE
A `PerformInteraction` response does not return a `choiceID` if the choice set was created with a static icon

### ISSUE THREAD

**Issue by NicoleYarroch** [smartdevicelink/sdl_hmi#222]
### Bug Report
A `PerformInteraction` response does not return a `choiceID` if the choice set was created with a static icon.

##### Reproduction Steps
1. Send a `CreateInteractionChoiceSet` with a single choice. The choice should have an `image` with an `imageType` of `STATIC`. A `CreateInteractionChoiceSet` will be returned with a `resultCode` of `SUCCESS`.
    ```
    Send RPC CreateInteractionChoiceSet (request), id: 4
    {
    choiceSet =     (
        "{\n    choiceID = 1;\n    image = \"{\\n    imageType = STATIC;\\n    isTemplate = 1;\\n        value = 0x7D;\\n}\";\n    menuName = one;\n    vrCommands =     (\n        one\n    );\n}"
    );
    interactionChoiceSetID = 1;
    }
    ```

2. Send a `PerformInteraction` request with the choice from step 1. The `interactionMode` should be `MANUAL_ONLY`.
    ```
    Send RPC PerformInteraction (request), id: 11
    {
    initialText = "Testing Manual";
    interactionChoiceSetIDList =     (
        1
    );
    interactionLayout = "LIST_ONLY";
    interactionMode = "MANUAL_ONLY";
    timeout = 10000;
    }
    ```

3. When the choice set appears on the HMI, select the only item in list (i.e. the cell with menu name "one" from step 1)

##### Expected Behavior
The `PerformInteraction` response returns the `choiceID` of the selected choice. 

##### Observed Behavior
The `PerformInteraction` response returns a `success` of `1` and a `resultCode` of `UNSUPPORTED_RESOURCE`. No `choiceID` is returned.
```
Message received: PerformInteraction (response), id: 11
{
    info = "Unsupported STATIC type. Available data in request was processed.";
    resultCode = "UNSUPPORTED_RESOURCE";
    success = 1;
}
```

##### OS & Version Information
* SDL HMI Version: 5.1.0 (develop branch)
* SDL Core Version: 5.1.3 (develop branch)
* Testing Against: The SDL iOS Swift example app.

##### Test Case, Sample Code, and / or Example App
https://github.com/smartdevicelink/sdl_ios/tree/master/Example%20Apps/Example%20Swift

**Comment by iCollin** [comment]
@NicoleYarroch Please confirm #223 fixes this issue for you.

**Comment by iCollin** [comment]
Closed with merge of #223
